### PR TITLE
iCubTemplateV6

### PR DESCRIPTION
**Pull request by davidetome**
This is a draft PR to discuss changes in order to document the migration to motor control v6.

@valegagge the `body_part--ebX-jA_B-mc.xml` was already aligned to [body_part--ebX-jA_B-mc.xml.txt](https://github.com/icub-tech-iit/sw-distro-management/files/3892977/body_part--ebX-jA_B-mc.xml.txt) a part of some comments added in the first commit.

I just added the groups for each board described in [Tutorial.txt](https://github.com/icub-tech-iit/sw-distro-management/files/3892970/Tutorial.txt)

cc @pattacini @ale-git

### CODE REVIEW

File: iCubGenova09/icub_wbd.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
     <devices>
     <!-- cartesian --> 
+
```
Comment (pattacini) [inline iCubGenova09/icub_wbd.xml:12]: Can we revert this change @davidetome ?

Comment (davidetome) [reply iCubGenova09/icub_wbd.xml:12]: Since there was this conflict I kept the version on the `devel` branch

Comment (pattacini) [reply iCubGenova09/icub_wbd.xml:12]: Uhm, you should have retained the other version as `devel` doesn't have the extra empty line.
It's nitpicking of mine but I think is important to confine the mods within the `iCubTemplates` directoryl

Comment (pattacini) [reply iCubGenova09/icub_wbd.xml]: Thanks!

**Comment by pattacini** [comment]
Hi @valegagge 

Could you please review the PR?

**Comment by davidetome** [comment]
just spoke w/ @valegagge , fixed the template as decided, we'll go on w/ the wiki mods

**Comment by valegagge** [comment]
Perfect!
I still have some doubts about the velocity PID configuration. I'd like to deepen with @ale-git.

**Comment by valegagge** [comment]
> Ciao @valegagge. please take a look at the diagrams [Presentation3.pptx](https://github.com/robotology/robots-configuration/files/8266815/Presentation3.pptx)

Great! We checked the diagrams together.

**Comment by valegagge** [comment]
After a chat with @ale-git, I updated the motor control config file accordingly[here](https://github.com/davidetome/robots-configuration/pull/1).
@davidetome if you are agree, you can merge [my PR](https://github.com/davidetome/robots-configuration/pull/1) and then merge this PR.

**Review by pattacini — APPROVED** [review]
Thanks heaps @davidetome and @valegagge for this contribution!
